tracer/build/_build/Build.Steps.cs

```diff
@@ -586,9 +588,32 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
             CompressZip(SymbolsDirectory, WindowsSymbolsZip, fileMode: FileMode.Create);
         });
 
+    Target ExtractDebugInfoAndStripSymbols => _ => _
+        .After(PublishProfilerLinux, PublishNativeLoaderLinux, PublishNativeProfilerLinux)
+        .OnlyWhenStatic(() => IsLinux)
+        .Executes(() =>
+        {
+            var directory = new DirectoryInfo(MonitoringHomeDirectory);
+            var files = directory.GetFiles("*.so", SearchOption.AllDirectories);
+
+            EnsureExistingDirectory(SymbolsDirectory);
+
+            foreach (var file in files)
+            {
+                var outputFile = Path.Combine(SymbolsDirectory, Path.GetFileNameWithoutExtension(file.Name));
```

Comment:
Just FYI, in Nuke they overloaded the `/` operator, so you can do something like this (which is kinda nice)
```suggestion
                var outputFile = SymbolsDirectory / Path.GetFileNameWithoutExtension(file.Name);
```